### DISCUSSION TITLE
Auto Reset Sliders to Background Profile

### DISCUSSION THREAD

**Discussion by michiganroaster** [artisan-roaster-scope/artisan#921] [General]
Is there a feature that forces Sliders to immediately/auto be set back to the background profiles starting positions? Trying to simplify the process for new production roasters so they don't forget to reset the sliders. 

Thank you for any help

**Comment by MAKOMO** [comment]
"Playback Events" is exactly doing this. You find it under menu Roast >> Background, first tab. However, once this feature is on it will replay all events from CHARGE on, either by time or temp. You can either manually turn it off after CHARGE (a shortcut is just pressing the j key; the background profiles name is set in red while replay is active), or define an alarm rule with action "Playback OFF" to fire some seconds after CHARGE.
